internal/commands/result.go

```diff
@@ -1284,8 +1377,21 @@ func exportJSONResults(targetFile string, results *wrappers.ScanResultsCollectio
 func exportJSONSummaryResults(targetFile string, results *wrappers.ResultSummary) error {
 	var err error
 	var resultsJSON []byte
+	var resultsToReport *wrappers.ResultSummary
 	log.Println("Creating summary JSON Report: ", targetFile)
-	resultsJSON, err = json.Marshal(results)
+
+	// Remove SCS Result if it exists
+	_, scsExists := results.EnginesResult[commonParams.ScsType]
```

Comment:
We need this behavior because we don't want to support the summaryJSON format yet. We include our results in this results object because we need it to present our counts on the console summary. So, for the summaryJSON we make a copy of results without our results in results.EnginesResult and report it in summaryJSON format.